### ISSUE TITLE
MS Viewer: Item Details/Digital Edition Rendering

### ISSUE THREAD

**Issue by awisnicki** [livingstoneonline/livingstoneonline#33]
1) Text Sizes

**Main title**
font-family: Crimson Text, serif
font-size: 2.3 em
font-weight: 600
margin-bottom: 10px
Note: should be flush left with the rest of the text, not hanging out into the margin, so I think margin-left: 10px

**Main text**
font: Source Sans Pro, sans-serif
font-size: .9 em

2) Combine "Publication" and "Digital Edition" into a single section called "Digital Edition" that begins before the line for "Directors". I'll provide the text for this section in a separate Github ticket.

3) **Section title for Digital Edition**
font-family: Crimson Text, serif
font-size: 1.5 em
Note: should be flush left with the rest of the text, not hanging out into the margin, so I think margin-left: 10px
Note: there should not be an empty line after this title. The text can be right below it.

**Comment by awisnicki** [comment]
All looks good except please adjust further as follows:

1) Text sizes
**Main title**
font-size: 1.75 em
margin-bottom: it appears that the solution I proposed (10em) does not work. The point is that right now there is an extra amount of space between the title and the items below it. The space below the title should be the same as the space between the different lines of this section.
Also, let's put a horizontal line below the title that runs end to end (left to right) and is gray. See how we do it on any page of the site, for instance: http://livingstone.lib.umd.edu/about-site/livingstone-online-introduction

3) **Section Title for Digital Edition**
font-size: 1.75 em
font-weight: 600
Also, let's put a horizontal line below the title that runs end to end (left to right) and is gray. See how we do it on any page of the site, for instance: http://livingstone.lib.umd.edu/about-site/livingstone-online-introduction
Finally, we need an empty line above this to set off this section from the previous one. Please make the empty line the size of a regular line in the section (rather than the size of the title).

In other words, I'd like to do the same thing for the Item Details and Digital Edition titles.

**Comment by awisnicki** [comment]
I'm closing out section #34 and combining it with one since it all goes together.

Still a touch of tweaking left on the Item Details and Digital Edition sections left as follows:
1. The vertical space between Item Details/Digital Edition and the gray line beneath is too large in each case and needs to be reduced. The vertical distance from these titles to the gray line, should be the same as from the gray line to the first line of text below.
2. We still need some vertical space about "Digital Edition" so that the "Digital Edition" section is clearly set off from the "Item Details" section.
3. The "Image Credits" section seems to have the word "Images" hard coded, so each version of this starts off with the word "Images" unnecessarily duplicated:
   **Image Credits:** Images Images [...]
4. In the "Directors" line, I meant for you to use "Leadership:" instead of "Directors", but I see now that "Directors" works just fine, so please just remove the word "Leadership:" (including the semi-colon that follows that word. In other words, the "Directors" line should read:
   **Directors:** Adrian S. Wisnicki (director), Christopher Lawrence (director emeritus), Megan Ward (associate director), Anne Martin (associate director)
5. The "Cite Item (MLA)" section needs to end with a final period (after the auto-generated date). Currently that period is missing.

**Comment by nigelgbanks** [comment]
> The vertical space between Item Details/Digital Edition and the gray line beneath is too large in each case and needs to be reduced. The vertical distance from these titles to the gray line, should be the same as from the gray line to the first line of text below.

Done.

> We still need some vertical space about "Digital Edition" so that the "Digital Edition" section is clearly set off from the "Item Details" section.

Margin is now set to 2em (was 0.5em).

> The "Image Credits" section seems to have the word "Images" hard coded, so each version of this starts off with the word "Images" unnecessarily duplicated: Image Credits: Images Images [...]

Hard coding of "Images" has been removed.

> In the "Directors" line, I meant for you to use "Leadership:" instead of "Directors", but I see now that "Directors" works just fine, so please just remove the word "Leadership:" (including the semi-colon that follows that word. In other words, the "Directors" line should read: Directors: Adrian S. Wisnicki (director), Christopher Lawrence (director emeritus), Megan Ward (associate director), Anne Martin (associate director)

Done.

> The "Cite Item (MLA)" section needs to end with a final period (after the auto-generated date). Currently that period is missing.

Done.

**Comment by awisnicki** [comment]
Everything looks great (thank you!), except please make one final change. Please change margin above "Digital Edition" to 1.5em and then please feel free to close this ticket.

**Comment by nigelgbanks** [comment]
Deployed to stage

**Comment by awisnicki** [comment]
Perfect. Thank you!
